glossary.html

```diff
@@ -33,10 +33,10 @@ <h1>Acronym Glossary</h1>
             </dl>              
        </main>
        
-       <Footer>
-       <hr>
+        <footer>
```

Comment:
Best to be consistent in your HTML formatting: either all uppercase or all lowercase tag names